tutor/templates/build/forum/Dockerfile

```diff
@@ -48,5 +48,7 @@ ENV MONGODB_AUTH ""
 ENV MONGOID_AUTH_MECH ""
 ENV MONGODB_HOST "mongodb"
 ENV MONGODB_PORT "27017"
+ENV MONGODB_PROTOCOL "mongodb"
+ENV MONGODB_PARAMS ""
```

Comment:
I don't understand what this `MONGODB_PARAMS` is supposed to do anymore, as it's not being defined anywhere.

Reply:
This was for additional parameters that may be needed for the connection to the cluster. I have removed it for now